mmv1/third_party/terraform/services/vmwareengine/resource_vmwareengine_subnet_test.go

```diff
@@ -58,6 +58,7 @@ resource "google_vmwareengine_network" "subnet-nw" {
 resource "google_vmwareengine_private_cloud" "subnet-pc" {
   location    = "%{region}-a"
   name        = "tf-test-subnet-pc%{random_suffix}"
+  type        = "TIME_LIMITED"
```

Comment:
This test is currently skipped in VCR: https://github.com/GoogleCloudPlatform/magic-modules/blob/e9f61669ae84e91231de5bc778c8d7ffd24dd074/mmv1/third_party/terraform/services/vmwareengine/resource_vmwareengine_subnet_test.go#L11

I'm not sure if it was skipped just to make VCR pass easier (in which case this might mean we could un-skip it) or if there was some non-deterministic behavior. What do you think? Either way this is a good change.

Reply:
Hey Stephen,
It was the former one, and I have unskipped the test now :)